docs/basics/DeepRLTutorial/TutorialDeepRL.rst

```diff
@@ -0,0 +1,605 @@
+Deep Reinforecement Learning in rlberry
+=======================================
+
+In this tutorial we will focus on Deep Reinforcement Learning with an
```

Comment:
I think the name is in the wrong order, it should be: "In this tutorial, we will focus on Deep Reinforcement Learning with the **Advantage Actor-Critic** algorithm."